styles/layout.less

```diff
@@ -1044,6 +1044,40 @@ table.dirlist {
     }
 }
 
+/* ------------------------------------------------------------------------ */
+/* Quizzes */
+
+@quiz-passed: #ccffcc;
+@quiz-not-passed: #ffcccc;
+@quizdate-ok: #ccffcc;
```

Comment:
Any particular reason why this isn't `@quiz-date-ok`?

Reply:
😆 -- None except an ancient irrational attempt to minimize the number of characters and avoid duplicate names. In light of some of the earlier changes, I've updated those two, also.